src/shm.rs

```diff
@@ -21,8 +30,9 @@ pub use crate::backend::shm::types::ShmOFlags;
 ///
 /// [POSIX]: https://pubs.opengroup.org/onlinepubs/9699919799/functions/shm_open.html
 /// [Linux]: https://man7.org/linux/man-pages/man3/shm_open.3.html
+#[doc(alias = "shm_open")]
 #[inline]
-pub fn shm_open<P: path::Arg>(name: P, flags: ShmOFlags, mode: Mode) -> io::Result<OwnedFd> {
+pub fn open<P: path::Arg>(name: P, flags: OFlags, mode: Mode) -> io::Result<OwnedFd> {
```

Comment:
What would you think of doing something like `use crate::shm;` at the top of the file here, and then changing `OFlags` to `shm::OFlags`, so that users always see the `OFlags` presented to them as `shm::OFlags`?